reworked loader ASCII art

hadfl opened the pull request:
- blank line between OmniOS logo and Menu inserted
- reworked OmniOS logo ASCII art
- reworked dragon ASCII art

danmcd commented:
Can you please:

1.) Put a screenshot up of what it looks like?

2.) Confirm nothing weird happens w.r.t. the bottom of the menu box?  My experiments were such that the bottom of the menu box would disappear in certain cases.  Check the submenus... all of them.

hadfl commented:
1. screenshots:
[loader.zip](https://github.com/omniti-labs/illumos-omnios/files/747817/loader.zip)

2. the "problem" that the bottom of the menu box disappears is due to the autoboot counter. when i moved the menu, i moved the autboot counter accordingly: https://github.com/omniti-labs/illumos-omnios/pull/9/files#diff-61cd34aa60b3158357cd371c9687eeceR63
